src/transforms/rename.jl

```diff
@@ -0,0 +1,37 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+  Rename(:col₁ => :newcol₁, :col₂ => :newcol₂, ..., :col₁ => :newcolₙ)
+
+The transform that renames `col₁` to `newcol₁`, `col₂` to `newcol₂`, ...
+"""
+struct Rename <: Stateless
+  names::Dict{Symbol,Symbol}
+end
+
+function apply(transform::Rename, table)
+  oldnames = Tables.columnnames(table)
+  newnames = map(oldnames) do oldname
+    if oldname in keys(transform.names) 
+      transform.names[oldname]
+    else
+      oldname
+    end
+  end
+  acols = [i for i in Tables.columns(table)]
+  𝒯 = (; zip(newnames, acols)...) |> Tables.materializer(table)
+  𝒯, nothing
+end
+
+function revert(transform::Rename, table)
+  # reversing the key-value pairs of the Dict
+  new_names = Dict()
+  for (new, old) in transform.names
```

Comment:
I think you you misplaced `new` and `old` in this loop. Can you please add tests to the transform so that we catch basic bugs?

Reply:
I will add tests (might take a little of poking around given this is my first forte into package testing.)
But I have been testing continuously while developing (I don't feel comfortable while writing code otherwise) and it works perfectly. Both reverting and appling invalid oldnames are handled as expected.
I can see why you said they might be misplaced, semantically I should probably swap them both in the for loop and the dict assignment (line32).

Reply:
Are you familiar with Julia's testing infrastructure? All you need to do is activate the folder of the package and type `] test`. You then add more tests and rerun.

Reply:
> Are you familiar with Julia's testing infrastructure?

No

> All you need to do is activate the folder of the package and type `] test`

that sounds like a simple workflow. I have used the package manager a lot but as a user, never as a package dev.
I can see the simple format in `test/transforms.jl` and I can add to that. 
Question: is there a way to just run the tests I'm developing instead of everything in there?

Reply:
Never used but I think Retest.jl has this functionality: https://github.com/JuliaTesting/ReTest.jl